packages/odyssey-react/src/components/TextInput/TextInput.module.scss

```diff
@@ -29,7 +87,7 @@
     }
 
     [dir="rtl"] & {
-      background-position: right $spacing-s center;
+      background-position: right var(--RtlBackgroundPositionRightOffset) center;
```

Comment:
Huh. It's weird that we weren't specific about this for the `ltr` default. I expected this variable to be something like `BackgroundPositionOffset`, used in two places with the `background-position` property varying between `right` and `left` for RTL/LTR.

What am I missing about my own CSS? 😬

Reply:
Looking closer this whole rule set isn't used anymore and can be safely removed. It only applies to the previous implementation where the search icon was a background image.